Update network-watcher-monitor-with-azure-automation.md

jboeshart opened the pull request:
Several updates here:
 - Made all cmdlet parameters into variables and listed them at the top of the script for ease of customization
 - Updated the text for $body to include the summary of the issue if one exists and set it to write out to the log
 - Updated the output of the healthy check to output $result.code instead of $result.connectionStatus, as the latter is not a valid option from the result of Start-AzureRmNetworkWatcherResourceTroubleshooting

azurecla commented:
Hi __@KylieLiang__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

tysonn commented:
No CLA needed, contributor is an employee.

tysonn commented:
Reviewed - I sent you a list of a few things to fix; once those items are sorted, I can accept the pull request.

tysonn commented:
Great, thanks for making the updates. This article should be available live around 11 AM on Wednesday (Pacific Time)

jboeshart committed:
Update network-watcher-monitor-with-azure-automation.md

Several updates here:
 - Made all cmdlet parameters into variables and listed them at the top of the script for ease of customization
 - Updated the text for $body to include the summary of the issue if one exists and set it to write out to the log
 - Updated the output of the healthy check to output $result.code instead of $result.connectionStatus, as the latter is not a valid option from the result of Start-AzureRmNetworkWatcherResourceTroubleshooting
